src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -34,19 +34,29 @@ public function normalize($object, $format = null, array $context = array())
         foreach ($object as $violation) {
             $violations[] = array(
                 'propertyPath' => $violation->getPropertyPath(),
-                'message' => $violation->getMessage(),
-                'code' => $violation->getCode(),
+                'type' => sprintf('urn:uuid:%s', $violation->getCode()),
```

Comment:
`Symfony\Component\Validator\ConstraintViolationInterface::getCode()` is nullable, so this format might currently result into `urn:uuid:`. Does the RFC 4122 states anything about such cases? Just set `null` instead?

Reply:
This part isn't covered by the RFC, I just try to mimic it (for consistency) in this extension. I've updated the code to not include the `type` key if no UUID is provided.